src/app/modules/private/private/private.component.html

```diff
@@ -0,0 +1,3 @@
+<lib-page-layout [sideMenuData]="menudata" (sideMenuClick)="selectedMenu($event)"></lib-page-layout>
+
+<lib-headers [logindetails] ="userDetails" (logoutClick)="logoutMethod($event)"></lib-headers>
```

Comment:
headers should be inside lib-page-layout right?

Reply:
yes,That is to pass data and to trigger logout